src/compat/compat.h

```diff
@@ -213,6 +213,72 @@ get_vacuum_options(const VacuumStmt *stmt)
 		   (analyze ? VACOPT_ANALYZE : 0);
 }
 
+/*
+ * The number of arguments of pg_md5_hash() has changed in PG >= 14.2. At the
+ * moment of writing this used to be an upcoming release, thus PG_VERSION_NUM
+ * couldn't be used to distinguish between REL_14_1 and REL_14_STABLE.
```

Comment:
Does this mean we're going to remove the copy of the function when the 14.2 is released?

Reply:
Sure, we can do it if we want to. Although so far we never used #if's for minor versions. If that's not some sort of policy, I see no reason to keep a copy of the code.